### PR TITLE
[LPX-657] Update moralis usage to fit Moralis 2.0

### PR DESCRIPTION
**Pull request by RuiLiu3**
Changes:
Update Moralis api from v1 to v2

Test:
curl api test

### CODE REVIEW

File: src/routes/nftRoutes.ts [hunk 1]
```diff
@@ -166,20 +166,10 @@ export async function getMoralisNftContractMetadataDict(
...
-          ErrorCode.MORALIS_REQUEST_FAILED,
-          ErrorMessage.MORALIS_REQUEST_FAILED,
-          moralisResponse.message,
-          ErrorType.DATABASE,
-          true,
-        ).toString();
-        logger.failed(errorMessage);
-        throw Error(`Moralis request failed with message: ${moralisResponse.message}`);
-      }
+      const moralisResponse = await Moralis.EvmApi.nft.getNFTMetadata(options);
```
Comment (pjxthy) [inline src/routes/nftRoutes.ts:172]: Why not keep the err handling logic?

File: src/routes/nftRoutes.ts [hunk 2]
```diff
@@ -219,22 +209,11 @@ export async function getMoralisNftTokenMetadataDict(
     const { contractAddress, tokenId } = tokenInfoFromFullTokenId(fullTokenId);
     const options = {
       address: contractAddress,
-      token_id: tokenId,
+      tokenId: tokenId,
       chain,
     };
     try {
-      const moralisResponse = await Moralis.Web3API.token.getTokenIdMetadata(options);
-      if (moralisResponse.message) {
```
Comment (pjxthy) [inline src/routes/nftRoutes.ts:227]: Ditto

File: src/routes/walletRoutes.ts [hunk 3]
```diff
@@ -26,11 +26,11 @@ const getWalletRoute: RouteGenerator = ({ responseCache }: AppDependency) => {
     }
     const options = {
       address: walletAddress,
-      chain: "eth",
+      chain: "0x1",
```
Comment (pjxthy) [inline src/routes/walletRoutes.ts:29]: Move this to constants and define readable chain names?

**Review by pjxthy — CHANGES_REQUESTED** [review]

File: src/routes/nftRoutes.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import { ethers } from "ethers";
-import { Moralis } from "./constants";
+import Moralis from 'moralis';
```
Comment (pjxthy) [inline src/routes/nftRoutes.ts:2]: Double quote for consistency?

File: src/routes/nftRoutes.ts [hunk 5]
```diff
@@ -166,20 +166,10 @@ export async function getMoralisNftContractMetadataDict(
     const options = {
       address: contractAddress,
       chain: "eth",
+      tokenId: "1"
```
Comment (pjxthy) [inline src/routes/nftRoutes.ts:169]: Not readable

**Review by pjxthy — CHANGES_REQUESTED** [review]

File: src/routes/nftRoutes.ts (hunk 1 above)
Comment (RuiLiu3) [reply src/routes/nftRoutes.ts:172]: the moralisResponse adaptor data type do not has the field message. TypeScript will report error when trying to access moralisResponse.message. 
.message field is used to response error message by Moralis. When moralis reponse error with bad http code, the promise will failed. We have the try catch code thus error could be logged in the catch block.

**Review by pjxthy — APPROVED** [review]

**Pull request merged**